src/lib/Scenes/Artwork/Components/ImageCarousel/FullScreen/ImageCarouselCloseButton.tsx

```diff
@@ -0,0 +1,56 @@
+import { CloseIcon } from "@artsy/palette"
+import { screenSafeAreaInsets } from "lib/utils/screenSafeAreaInsets"
+import { observer } from "mobx-react"
+import React, { useContext } from "react"
+import { TouchableOpacity, View } from "react-native"
+import { ImageCarouselContext } from "../ImageCarouselContext"
+import { boxShadow } from "./boxShadow"
+
+// taken from https://github.com/artsy/eigen/blob/0831853cb574566415f3bd8b3908b26b61f61eec/Artsy/View_Controllers/Util/ARNavigationController.m#L125
+const CLOSE_BUTTON_MARGIN = 12
+
+export const ImageCarouselCloseButton: React.FC<{ onClose(): void }> = observer(({ onClose }) => {
+  const {
+    state: { fullScreenState },
+  } = useContext(ImageCarouselContext)
+  const showCloseButton = fullScreenState === "entered" || fullScreenState === "animating entry transition"
+  return (
+    <View
+      style={{
+        position: "absolute",
+        left: 0,
+        top: screenSafeAreaInsets.top,
+      }}
+    >
+      <TouchableOpacity onPress={onClose}>
+        <View
+          style={{
+            width: 40,
+            height: 40,
+            paddingLeft: CLOSE_BUTTON_MARGIN,
+            paddingTop: CLOSE_BUTTON_MARGIN,
+            paddingRight: 20,
+            paddingBottom: 20,
+          }}
+        >
```

Comment:
Can this be `<Box>`? Under the hood we use `View`

Reply:
I've used View in a bunch of places where I could have used Flex and Box. This was mainly to keep the time between tapping an image and showing the full screen carousel as short as possible. styled-components adds some overhead when computing and applying styles, which doesn't seem worth the DX of shorthand property names to me.